packages/uikit-react-native/src/components/ChannelInput/index.tsx

```diff
@@ -211,6 +223,20 @@ const styles = createStyleSheet({
   inputDefault: {
     height: 56,
   },
+  input: {
+    flex: 1,
+    marginRight: 4,
+    borderRadius: 20,
+    paddingTop: 8,
+    paddingBottom: 8,
+    minHeight: 36,
+    // Android - padding area is hidden
+    // iOS - padding area is visible
+    maxHeight: Platform.select({ ios: 36 * 2 + 16, android: 36 * 2 }),
```

Comment:
When padding is applied to the top/bottom of TextInput, on Android, the text may be obscured by the padding area, whereas on iOS, it remains visible.
This results in a discrepancy in the number of visible text lines on the UI (e.g., three lines displayed on Android versus only two on iOS).
To address this, add an additional offset equal to the padding vertical.